apps/mobile/src/shared/constants.ts

```diff
@@ -1,4 +1,16 @@
+import { LEATHER_GUIDES_URL } from '@leather.io/constants';
+
 export const HEADER_HEIGHT = 64;
 
 export const GITHUB_ORG = 'leather-io';
 export const GITHUB_REPO = 'mono';
+
+export const LEATHER_GUIDES_MOBILE_ANALYTICS = `${LEATHER_GUIDES_URL}/mobile-analytics`;
+export const LEATHER_GUIDES_MOBILE_ACCOUNT_IDENTIFIER = `${LEATHER_GUIDES_URL}/mobile-account-identifier`;
+export const LEATHER_GUIDES_MOBILE_BITCOIN_UNIT = `${LEATHER_GUIDES_URL}/mobile-bitcoin-unit`;
+// TODO: Add this to Leather Guides on Framer
+export const LEATHER_GUIDES_MOBILE_APP_AUTHENTICATION = `${LEATHER_GUIDES_URL}/mobile-app-authentication`;
```

Comment:
I added this URL for `LEATHER_GUIDES_MOBILE_APP_AUTHENTICATION` but the link is not live yet. I passed on the URL so it can be added later

Reply:
`LEATHER_GUIDES_MOBILE_REMOVE_WALLET` is also not yet live but I added it